### ISSUE TITLE
Incorrect aria-expanded attribute

### ISSUE THREAD

**Issue by svossen** [inthepocket/cookie-though#751]
Hi, I have a small accessibility improvement.

The element `<div class="ct-collapse">` has an `aria-expanded` attribute. This attribute should be used on a control to indicate if its controlled elements are hidden or displayed. 

So the `<button class="ct-customization-button">` should have the `aria-expanded` attribute, and `<div class="ct-collapse">` should have an `aria-hidden` attribute.

Google Lighthouse also indicates the current implementation is in violation with the specs.
